### PR TITLE
Port tweakfedpegscript from liquid-deamon

### PR DESCRIPTION
**Pull request by mword**
This is a port of the changes made in liquid-deamon to add tweakfedpegscript in rpc/misc.cpp.

### CODE REVIEW

File: qa/rpc-tests/pegging.py [hunk 1]
```diff
@@ -168,6 +168,7 @@ def test_pegout(parent_chain_addr, sidechain):
     bitcoindstart = bitcoin_bin_path+"/bitcoind -datadir="+bitcoin_datadir
     subprocess.Popen(bitcoindstart.split(), stdout=subprocess.PIPE)
 
+
```
Comment (instagibbs) [inline qa/rpc-tests/pegging.py]: unrelated line

File: src/rpc/misc.cpp [hunk 2]
```diff
@@ -593,11 +625,11 @@ static const CRPCCommand commands[] =
     { "util",               "createblindedaddress",   &createblindedaddress,   true,  {} },
     { "util",               "verifymessage",          &verifymessage,          true,  {"address","signature","message"} },
     { "util",               "signmessagewithprivkey", &signmessagewithprivkey, true,  {"privkey","message"} },
-
+    { "util",               "tweakfedpegscript",      &tweakfedpegscript,      true,  {"claim_script"} },
     /* Not shown in help */
     { "hidden",             "setmocktime",            &setmocktime,            true,  {"timestamp"}},
     { "hidden",             "echo",                   &echo,                   true,  {"arg0","arg1","arg2","arg3","arg4","arg5","arg6","arg7","arg8","arg9"}},
-    { "hidden",             "echojson",               &echo,                  true,  {"arg0","arg1","arg2","arg3","arg4","arg5","arg6","arg7","arg8","arg9"}},
+    { "hidden",             "echojson",               &echo,                   true,  {"arg0","arg1","arg2","arg3","arg4","arg5","arg6","arg7","arg8","arg9"}},
```
Comment (instagibbs) [inline src/rpc/misc.cpp]: unrelated space

**Comment by instagibbs** [comment]
ACK otherwise

**Comment by jtimon** [comment]
Perhaps it would be nice to wait for https://github.com/ElementsProject/elements/pull/362 ?
Otherwise #362 could be rebased on top of this.
My only point is that pegging.py and feature_2wp.py should be mirrored in tests until the latter fully replaces the former after adding a --custom-binary option or similar, to support testing with any bitcoin core version as parent chain (like pegging currently does).

File: qa/rpc-tests/pegging.py (hunk 1 above)
Comment (mword) [reply qa/rpc-tests/pegging.py]: Cleaned this up

File: src/rpc/misc.cpp (hunk 2 above)
Comment (mword) [reply src/rpc/misc.cpp]: Re-added the line that was removed: The space change was just to fix the column alignment in this line.

**Comment by mword** [comment]
@jtimon this is already implemented in liquid so when #362 is ported to liquid it will be rebased on top of this change there. Which is less confusing?

**Comment by instagibbs** [comment]
If I had to guess I'd say it would be simpler to merge this first and rebase the other PR, since it's still dependent on yet another PR.

**Comment by instagibbs** [comment]
Needs squash

**Comment by mword** [comment]
@instagibbs Squashed (I hope)

**Comment by instagibbs** [comment]
ACK
